BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -32,6 +33,63 @@ public class RealSense2 : Camera, IDisposable
         private bool _pipelineRunning = false;
         private float _depthScale = 1.0f;
 
+        #region Filter
+        public class RealSense2Filter
```

Comment:
Why not name the class just `Filter`. It's nested in the RealSense2 class already.